native/common/jp_classhints.cpp

```diff
@@ -324,16 +329,6 @@ class JPHintsConversion : public JPConversion
 	JPMatch::Type matches(JPClass *cls, JPMatch &match) override
 	{
 		auto *pyhints = (PyJPClassHints*) cls->getHints();
-		// GCOVR_EXCL_START
```

Comment:
the evil of code coverage exclusion hid the fixed bug?

Reply:
This code was moved to a new common location where it was unconditionally exercised.  I don't think there was a bug here other than hints only getting created occasionally and this code path or returning `_none` thus being the weird edge case that happen once in the whole execution.